src/Components/NavBar/NotificationsBadge.tsx

```diff
@@ -42,6 +42,12 @@ export const NotificationsBadge: React.FC<{
 
         let count = totalUnread
 
+        // User has no notifications; clear the cookie
```

Comment:
It seems like this and the branch it was swapped with are mutually exclusive, and order shouldn't matter. Is this move because you prefer to `return null` early, or am I misunderstanding the code?

Reply:
I originally had this branch further below and for clarity reasons moved it up to the top. No actual behavior has changed.